styleguide/components/21-atoms/field-description/field-description.twig

```diff
@@ -1,3 +1,6 @@
 {% if field_description %}
-  <div class="field-description">{{ field_description|default(field_description) }}</div>
+  <div class="field-description">
+    <h3 class="visually-hidden">{{ 'Field description' }}</h3>
```

Comment:
See before.
If used, note that we would have to be very sure about the heading order.
Are there UX specifications for the heading depth of forms?